src/demo/components/Icon/Icon.scss

```diff
@@ -0,0 +1,40 @@
+// stylelint-disable max-line-length
+$svg-data: 'data:image/svg+xml;base64';
+$svg-size: 16px;
+
+@mixin icon($icon) {
+  width: $svg-size;
+  height: $svg-size;
+  background-image: url('#{$svg-data},#{$icon}');
+  background-size: $svg-size $svg-size;
+}
+
+.iconDown {
+  $icon: 'PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDBWMHoiIGZpbGw9Im5vbmUiLz48cGF0aCBkPSJNMjAgMTJsLTEuNDEtMS40MUwxMyAxNi4xN1Y0aC0ydjEyLjE3bC01LjU4LTUuNTlMNCAxMmw4IDggOC04eiIvPjwvc3ZnPg==';
```

Comment:
For maintenance reasons, I'd prefer to keep here also legible source of the SVG. Now I'd like to make the icons inherit their color from text (leaving the `fill` unset or set to `currentColor`) which is hard without knowing the source image.

My suggestion is to use something like [this](https://gist.github.com/stowball/ff5803cb6cc466ac77ccc21c1ed19842) or [this](https://codepen.io/kevinweber/pen/dXWoRw).

Reply:
I tried it but without success. Can you try to do it instead of me? Icons are taken from Material Icons:

* sync => loading
* arrow_downward => down
* arrow_upward => up
* stars => star
* check_circle => success

Reply:
I succeeded but anyway — you can only have SVG inlined in order to inherit current text color (`fill: currentColor`). I added it as a fixup to your commit.